### PR TITLE
Htaq translate

### PR DESCRIPTION
**Pull request by artem-trubin**
Translated HTAQ.md file. There are some TODOs, for example where list of chats must be (since we don't have newly created chats yet) and Table Of Content must be regenerated, I'm not sure how to do it.

### CODE REVIEW

File: HTAQ.md [hunk 1]
```diff
@@ -25,90 +25,87 @@ _How To Get Help Like A Boss_
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 <!-- generated with [DocToc](https://github.com/thlorenz/doctoc) -->
 
-## INTRO
-In the world of hackers, the kind of answers you get to your technical questions depends as much on the way you ask the questions as on the difficulty of developing the answer. This guide will teach you how to ask questions in a way more likely to get you a satisfactory answer.
+## ВСТУП
+У світі хакерів, відповіді, які ви отримаєте на свої технічні запитання, настільки ж залежать від того як саме ви задасте своє питання, як і від складності формулювання цієї відповіді. Цей гайд навчить вас як задавати питання так, щоб з найбільшим шансом отримати відповідь яка вас задовольнить.
 
-**It sometimes looks like we're reflexively rude to newbies and the ignorant. But this isn't really true. Please try to ask good questions :smile_cat:**
+**Іноді може здаватись що ми відштовхуюче грубі з новаками, або приділяємо їм мало уваги. Але насправді це зовсім не так. Будь ласка, спробуйте краще задати питання :smile_cat:**
```
Comment (A-Ostrovnyy) [inline HTAQ.md:31]: новачками

File: HTAQ.md [hunk 2]
```diff
@@ -25,90 +25,87 @@ _How To Get Help Like A Boss_
...
-It is important to give your audience the proper context.
-Luckily delivering context in software engineering
-is much simpler than in many other fields.
+Дуже важливо надавати своїй аудиторії належний контекст. На щастя, надання контексту у програмуванні набагато простіше ніж в інших сферах.
 
-_To get the best feedback on your issue,
-please, follow the structure of asking below.
-Of course wording can be different from suggested,
-depending on your particular situation_
+_Щоб надати зворотній зв'язок за вашим питанням, будь ласка, використовуйте структуру запитування, що наведена нижче. Авжеж, конкретні слова не повинні бути такими самими, все залежить від вашої ситуації._
```
Comment (A-Ostrovnyy) [inline HTAQ.md:37]: `структуру питання` or `структуру запитів`

File: HTAQ.md [hunk 3]
```diff
@@ -25,90 +25,87 @@ _How To Get Help Like A Boss_
...
+Коли ви задаєте питання, покажіть, що ви спочатку спробували ці методи; це допоможе показати, що ви не лінитесь та не гаєте свій час та час інших.
+Що навіть краще, покажіть чому ви навчились, виконуючи ці речі. Нам подобається відповідати на питання людей, які демонструють що вони зможуть навчитись з цих відповідей.
 
-**Remember, [there is no such thing as a stupid question.](https://en.wikipedia.org/wiki/No_such_thing_as_a_stupid_question)**:wink:
+Не поспішайте. Не очікуйте, що вдасться вирішити складну проблему парою секунд Гуглінгу. Розслабтесь, візміть чашку чаю та добряче подумайте над проблему перш ніж турбувати експертів. Повірте, вони зможуть по вашому питанню зрозуміти наскільки багато читання та розмірковування у вас пішло на задачу, та захочуть допомогти, якщо побачать що ви прийшли підготовленими. Не вистрілюйте весь свій арсенал питань просто тому що ваш перший пошуковий запит не видав вам жодних відповідей (або забагато відповідей).
```
Comment (A-Ostrovnyy) [inline HTAQ.md:60]: `подумайте над проблемою` or 'обдумайте проблему'

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!

**Review by suchov — APPROVED** [review]

**Comment by suchov** [comment]
asked @A-Ostrovnyy to review one more time

**Review by A-Ostrovnyy — APPROVED** [review]

**Comment by A-Ostrovnyy** [comment]
Awesome!

**Pull request merged**